build/cloudbuild-tf-apply.yaml

```diff
@@ -14,12 +14,16 @@
 
 timeout: 1200s
 steps:
-- name: gcr.io/$PROJECT_ID/terraform
+- id: 'setup'
+  name: gcr.io/$PROJECT_ID/terraform
   entrypoint: /bin/bash
   args:
   - -c
   - |
+    echo "Setting up gcloud for impersonation"
     gcloud config set auth/impersonate_service_account ${_TF_SA_EMAIL}
+    echo "Adding bucket information to backends"
+    for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/${_STATE_BUCKET_NAME}/' $i; done
```

Comment:
will this change conflict with the change `sed -i "s/UPDATE_ME/${_STATE_BUCKET_NAME}/g" ./envs/*/backend.tf` in PR https://github.com/terraform-google-modules/terraform-example-foundation/pull/100

Reply:
Agreed, we can probably consolidate this logic in the update script.

Reply:
i like the `find` method better.  I think at this point almost anything in PR #100 is deprecated